action.yml

```diff
@@ -20,6 +20,10 @@ inputs:
     description: tag for the created image
     required: false
     default: latest
+  namespace:
+    description: namespace for the application
+    required: false
+    default: default
```

Comment:
Should we specify a default value here (1) or keep this in porter CLI (2)?  

I had followed option 2 for the `host` input.

Reply:
Hmm, so the issue is that `namespace` is a flag that needs to be passed to the porter CLI, so we would need to add casing for whether we want to add the flag in the entrypoint script. It's not an optional environment variable like `host`. Do you think it's worth making the entrypoint script more complex to case on whether to add a flag or not?

Reply:
Probably not